client/app/services/catalogs-state.service.js

```diff
@@ -68,6 +68,134 @@
       return CollectionsApi.query('service_templates/' + serviceTemplateId + '/service_dialogs', options);
     };
 
+    catalogState.setCatalogServiceTemplates = function(catalog, serviceTemplates) {
+      if (!catalog.serviceTemplates) {
```

Comment:
No action: is this ever false or rather only ever undefined, if so prefer `angular.isUndefined()`